openedx/core/djangoapps/theming/core.py

```diff
@@ -47,7 +47,7 @@ def comprehensive_theme_changes(theme_dir):
 
     favicon = component_dir / "static" / "images" / "favicon.ico"
     if favicon.isfile():
-        changes['settings']['FAVICON_PATH'] = str(favicon)
+        changes['settings']['FAVICON_PATH'] = "images/favicon.ico"
```

Comment:
@ziafazal as per our discussion I have a created a separate issue [WL-307](https://openedx.atlassian.net/browse/WL-307) for this change to ease testing and verification.